### ISSUE TITLE
A strange problem in gst+speaker train

### ISSUE THREAD

**Issue by Adliyan** [espnet/espnet#3706]
I am training a fastspeech2 model with gst, but when initializing self.convs of the ReferenceEncoder class in style_encoder.py, the weight of the batchnorm2d layer in self.convs is automatically initialized to 0. I added the following code, and the weight becomes 1, but in the forward phase, it is initialized to 0 again. Have you encountered this problem,please?

```python
for m in self.convs:             
    if isinstance(m, torch.nn.BatchNorm2d):
        torch.nn.init.constant_(m.weight, 1)                
        torch.nn.init.constant_(m.bias, 0)
```
